manual/manual/cmds/instrumented-runtime.etex

```diff
@@ -0,0 +1,332 @@
+\chapter{Runtime tracing with the instrumented runtime}
+%HEVEA\cutname{instrumented-runtime.html}
+
+This chapter describes the OCaml instrumented runtime, a runtime variant
+allowing the collection of events and metrics.
+
+Collected metrics include time spent executing the {\em garbage collector},
+allowing to measure the overall execution time of individual pauses,
+down to the time spent in specific parts of the garbage collection algorithm.
+
+Insight is also given on memory allocation and motion, recording
+the size of allocated memory blocks, as well as value promotions from the
+{\em minor heap} to the {\em major heap}.
+
+\section{s:instr-runtime-overview}{Overview}
+
+Once compiled and linked with the instrumented runtime, any OCaml program
+can generate {\em trace files} that can then be read
+and analyzed by users in order to understand specific runtime behaviors.
+
+The {\em Common Trace Format} is the general purpose binary
+format used to encode traces.
+A complete trace is comprised of:
```

Comment:
```suggestion
A complete trace comprises of:
```

Reply:
comprises ~of~ or
~comprises~ consists of